transport/tchannel/handler_test.go

```diff
@@ -273,6 +274,17 @@ func TestResponseWriter(t *testing.T) {
 			arg2: []byte("{}\n"),
 			arg3: []byte("{}"),
 		},
+		{
+			format: tchannel.Raw,
+			apply: func(w *responseWriter) {
+				w.SetApplicationError()
```

Comment:
This wont be exposed directly to users right? They'll return a Thrift error instead, and we'll call this internally?